Library/Homebrew/cmd/desc.rb

```diff
@@ -0,0 +1,59 @@
+require 'descriptions'
+require 'cmd/search'
+
+USAGE =<<EOM
```

Comment:
Fair enough. I will make it behave like `brew deps` just raise a `FormulaUnspecifiedError` instead.